tests/config.toml

```
@@ -1,6 +1,6 @@
 key = "tests/key.pem"
 cert = "tests/cert.pem"
-root = "tests/root.pem"
+ca_certs = ["tests/ca_certs.pem"]
```

```suggestion
ca_certs = ["tests/ca_cert.pem"]
```

I'd like to suggest this small change. I think the file also needs renaming to `ca_cert.pem`.

---

As you pointed out, I modified the path name.